keycloak-dev/realms/moh_applications/user-management-service-integration-tests/versions.tf

```
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    keycloak = {
+      source  = "mrparkers/keycloak"
+      version = ">= 3.0.0"
```

If you can change this please, just for consistency!

```suggestion
      version = "3.9.1"
```